features/step_definitions/journey_test_steps.rb

```diff
@@ -1471,15 +1472,27 @@ def filter_to_check(filter_name,filter_value,filter_exist)
   page.should have_selector(:xpath, ".//*[@id='global-breadcrumb']/nav/*[@role='breadcrumbs']/li[1]//*[contains(text(), 'Admin home')]")
 end
 
-Then /I am presented with the G-Cloud 7 Statistics page$/ do
-  page.find(
-    :xpath,
-    "//p[contains(text(), 'G-Cloud 7')]/../h1[contains(text(), 'Statistics')]"
-    )
-    current_url.should end_with("#{dm_frontend_domain}/admin/statistics/g-cloud-7")
+Then /I am presented with the '(.*)' page$/ do |statistics_page|
+  case statistics_page
+  when 'G-Cloud 7 statistics'
+    page.find(
```

Comment:
Could this line come before the `case`, to avoid repeating the xpath logic?
e.g. (if `framework_name` is the parameter passed in)
```
page.find(
       :xpath,
       "//p[contains(text(), framework_name)]/../h1[contains(text(), 'Statistics')]"
       )
```